### ISSUE TITLE
Admin: Ne pas afficher les "organisations" sans jeton

### ISSUE THREAD

**Issue by skelz0r** [etalab/admin_api_entreprise#254]
Ça n'a aucun intérêt au niveau du dashboard. 

Ceci est possible depuis l'introduction de la gestion des webhooks DataPass, ce qui n'était pas le cas avant ( https://github.com/etalab/admin_api_entreprise/pull/141 )

Thoughts @DorineLam ?

**Comment by DorineLam** [comment]
Juste pour bien comprendre, c'est des orgas qui sont dans notre base, mais qui n'ont plus de jetons (= elles en ont eu un il fut un temps, mais elles n'en ont plus ?)

**Comment by skelz0r** [comment]
Non, ces "organisations" (qui sont en réalité des "users") sont les utilisateurs associés aux demandes d'authorisations sur DataPass. Maintenant qu'on persiste les users dès qu'ils créent les demandes (notamment pour envoyer des emails depuis notre dashboard), ces utilisateurs n'ont tout simplement pas de jetons.

Btw pour moi les utilisateurs sans jeton ne devraient pas existe sur notre backend (ils peuvent avoir des jetons expirés).

**Comment by Haelle** [comment]
on clos au profit de Metabase
